contracts/implementations/RMRKBaseStorageImpl.sol

```diff
@@ -16,8 +16,8 @@ import "../RMRK/access/OwnableLock.sol";
  */
 
 contract RMRKBaseStorageImpl is OwnableLock, RMRKBaseStorage {
-    constructor(string memory symbol_, string memory type__)
-        RMRKBaseStorage(symbol_, type__)
+    constructor(string memory metadataURI, string memory type_)
```

Comment:
Same question about the possibility of changing the `metadataURI` to ` metadataUri_` as above.